Kickstarter-iOS/Views/Cells/RewardCell.swift

```diff
@@ -6,6 +6,8 @@ import Prelude
 final class RewardCell: UICollectionViewCell, ValueCell {
   private let containerView = UIView(frame: .zero)
 
+  static let fixedCardWidth: CGFloat = 249
```

Comment:
Not based on device width?

Reply:
@justinswart no, chatted with Izzy about it and it seems the designs do imply that the card width should stay constant.